pipelineruns/build_pipelinerun_pac_merge_pass.yaml

```diff
@@ -6,13 +6,13 @@ metadata:
   annotations:
     "appstudio.redhat.com/updateComponentOnSuccess": "false"
     pipelinesascode.tekton.dev/on-target-branch: '[main]'
-    pipelinesascode.tekton.dev/repo-url: 'https://github.com/redhat-appstudio/build-service'
+    pipelinesascode.tekton.dev/repo-url:
     pipelinesascode.tekton.dev/sha-title: 'Merge pull request #35 from psturc/PLNSRVCE-281-vol2'
     pipelinesascode.tekton.dev/git-auth-secret: pac-gitauth-nthx
     pipelinesascode.tekton.dev/max-keep-runs: '5'
-    pipelinesascode.tekton.dev/sha-url: https://github.com/redhat-appstudio/build-service/commit/f13b881e3404ad3401fa7b533df3f7daddd11b55
+    pipelinesascode.tekton.dev/sha-url:
```

Comment:
Just an idea - we could create an example PR in one of our repos (or create a new one in the [hacbs-integration-tests](https://github.com/hacbs-integration-tests) org) and keep it open as an example. Then we could set these labels to point to that, but with an added comment to replace those values when testing from our own setups (as described [here](https://docs.google.com/document/d/1DhmxW2LW4S7dwxFBP7GFYz3TFy2AmzxeUa6rEKMrb_8/edit#bookmark=id.tkghlpuk9wz)).

Reply:
a great idea, working on creating it 
thanks @dirgim

Reply:
Yep, I'm fine with adding this as an additional AC.

Reply:
Let me know if you need help with setting up the repo or PR for this.

Reply:
Done, 
https://github.com/hacbs-integration-tests/Integration-pac-example